### PR TITLE
ts menubar

### PR DESCRIPTION
**Pull request by chrisnojima**

### CODE REVIEW

**Comment by chrisnojima** [comment]
@keybase/react-hackers this is ready

File: shared/menubar/out-of-date.tsx [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
 import * as React from 'react'
 import * as Kb from '../common-adapters'
 import * as Styles from '../styles'
 import * as ConfigTypes from '../constants/types/config'
 import flags from '../util/feature-flags'
 
 type Props = {
-  outOfDate?: ConfigTypes.OutOfDate,
-  updateNow?: () => void,
+  // eslint-disable-next-line no-use-before-define
```
Comment (buoyad) [inline shared/menubar/out-of-date.tsx]: Don't think this is needed

File: shared/menubar/user-add.desktop.tsx [hunk 2]
```diff
@@ -34,12 +34,12 @@ const UserButton = ({isPublic, onClick}: {isPublic: boolean, onClick: () => void
...
-  isPublic: boolean,
-  onSubmit: () => void,
-  onCancel: () => void,
-  onUpdateText: () => void,
-  username: ?string,
-  text: string,
+  isPublic: boolean
+  onSubmit: () => void
+  onCancel: () => void
+  onUpdateText: (string) => void
```
Comment (buoyad) [inline shared/menubar/user-add.desktop.tsx]: `string` is being interpreted as the name of the argument here

File: shared/menubar/out-of-date.tsx (hunk 1 above)
Comment (chrisnojima) [reply shared/menubar/out-of-date.tsx]: it is due to babel-eslint issues

**Pull request merged**